src/main/java/mate/academy/bookstore/dto/user/UserRegistrationRequestDto.java

```diff
@@ -0,0 +1,21 @@
+package mate.academy.bookstore.dto.user;
+
+import jakarta.validation.constraints.Email;
+import lombok.Data;
+import mate.academy.bookstore.validation.FieldMatch;
+import org.hibernate.validator.constraints.Length;
+
+@Data
+@FieldMatch(first = "password", second = "repeatPassword",
+        message = "The password fields don't match")
+public class UserRegistrationRequestDto {
+    @Email
```

Comment:
```suggestion
    @Email
    @NotBlank
```

Reply:
> 

Sorry im confused, first you commented to delete this annotation, i've deleted it,and now i have to add it?

Reply:
I removed those annotation for shippingAddress field only